hal/armv7m/imxrt/117x/imxrt117x.c

```diff
@@ -180,7 +180,8 @@ int _imxrt_setIOpad(int pad, char sre, char dse, char pue, char pus, char ode, c
 		return -EINVAL;
 	}
 
-	if ((pad >= pctl_pad_gpio_emc_b1_00) && (pad <= pctl_pad_gpio_disp_b2_15)) {
+	if (((pad >= pctl_pad_gpio_emc_b1_00) && (pad <= pctl_pad_gpio_emc_b2_20)) ||
```

Comment:
So, it's in fact like the most of them? Then perhaps invert this if and do it from `pctl_pad_gpio_ad_00` up to `pctl_pad_gpio_ad_35` instead?

Reply:
true, but there are still e.g. gpio SNVS and LPSR IOpads with numbers > gpio_sd_b1_00 that are configured like this, so the if would still look something like this:
```c
if (((pad >= pctl_pad_gpio_ad_00) && (pad <= pctl_pad_gpio_ad_35)) || (pad >= pctl_pad_test_mode)) {
    /* ... */
}
```

Reply:
Right, it's pretty much the same either way